client/rpc/src/eth.rs

```diff
@@ -1173,17 +1173,20 @@ impl<B, C, P, CT, BE, H: ExHashT> EthApiT for EthApi<B, C, P, CT, BE, H> where
 pub struct NetApi<B: BlockT, BE, C, H: ExHashT> {
 	client: Arc<C>,
 	network: Arc<NetworkService<B, H>>,
+	peer_count_as_hex: bool,
```

Comment:
Could you use your `pub enum PeerCount` here?

Reply:
Not really, in order to do that we'd need to provide the PeerCount variant from the caller end. Bool is good :)

Reply:
I like bool better. A `PeerCount` needs to have data included and then there is the choice of what dummy data to include.

We could use `PeerCount:U32(0)` in place of `true` and `PeerCount::String("".into())` in place of false, but then a reader of the code might incorrectly think that data means something.